### PR TITLE
Add meeting recordings for wg-multitenancy and fix charters

### PR DESCRIPTION
**Pull request by nikhita**
The links are taken from the meeting notes document: https://docs.google.com/document/d/1fj3yzmeU2eU8ZNBCUJG97dk_wC7228-e_MmdcmTNrZY/edit?usp=sharing.

/assign @jessfraz @davidopp

### CODE REVIEW

**Comment by jessfraz** [comment]
Great thanks!

/lgtm

**Comment by nikhita** [comment]
/assign @cblecker 
for approval

File: sigs.yaml [hunk 1]
```diff
@@ -1992,7 +1992,7 @@ workinggroups:
     dir: wg-multitenancy
     mission_statement: >
       Define the models of multitenancy that Kubernetes will support. Discuss and execute upon any remaining work that needs to be done to support these models. Create conformance tests that will prove that these models can be built and used in production environments.
-    charter_link:
+    charter_link: https://docs.google.com/document/d/1SkVdOPR4jozYDT8ro51hU3yrf1sHS8Gez73xM3PCsVo/edit
```
Comment (cblecker) [inline sigs.yaml]: Charters should be in-repo, so they can be versioned and approved by steering.

Comment (nikhita) [reply sigs.yaml]: Yup, thought so too. There was one more wg which had a google-doc charter. Removed it as well. :+1:

**Comment by nikhita** [comment]
cc @ant31 @bryanl @garethr for the app-def change

**Pull request merged**